packages/input/src/LionInput.js

```diff
@@ -7,12 +7,21 @@ import { LionField } from '@lion/field';
  * @extends LionField
  */
 export class LionInput extends LionField {
-  get delegations() {
+  static get properties() {
     return {
-      ...super.delegations,
-      target: () => this.inputElement,
-      properties: [...super.delegations.properties, 'readOnly'],
-      attributes: [...super.delegations.attributes, 'readonly'],
+      ...super.properties,
+      /**
+       * A Boolean attribute which, if present, indicates that the user should not be able to edit
+       * the value of the input. The difference between disabled and readonly is that read-only
+       * controls can still function, whereas disabled controls generally do not function as
+       * controls until they are enabled.
+       *
+       * (From: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-readonly)
+       */
+      readOnly: {
+        type: Boolean,
+        attribute: 'readonly',
+      },
```

Comment:
do we also want `reflect: true` for `readOnly `?

Reply:
In the end we might want to. But I think it would be adding functionality not there before (not related to this small bugfix). We can reconsider a proper solution once we do all delegation via a new convention/mixin